atom/app/node_main.cc

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by the MIT license that can be
 // found in the LICENSE file.
 
+#ifdef ENABLE_RUN_AS_NODE
```

Comment:
You have already excluded this file from the build in the `filenames.gypi`, haven't you?
Seems like you don't need this `#ifdef` here.

Reply:
@alexeykuzmin Just fyi, it's mostly a sanity check, in case someone included it in build and didn't set the define. In the `.h` file, you can accidentally `#include` it without it itself being part of build.

Reply:
Then you should have used `NOTREACHED()` under a `#ifndef ENABLE_RUN_AS_NODE` check =/

Reply:
All developer mistakes should be punished by failing builds or tests =)